utmutatok/fullstack/fullstackdeveloper.md

```diff
@@ -0,0 +1,232 @@
+# Fullstack (web) developer, 2019
+
+## Egy falatnyi történelem 
+
+A webes, illetve a webes alkalmazásfejlesztés hatalmasat fejlődött az elmúlt akár 5 évben, sőt, mióta léteznek modern webalkalmazásokhoz használatos framework-ök, komoly változás következik be akár minden hatodik hónapban. Ilyenkor természetesen, illetve jobb esetben, a fejlesztőre van bízva, hogy ezeket mennyire, és milyen intenzitással hajlandó követni. 
+
+## Mi az a fullstack fejlesztő? 
+
+A fullstack fejlesztő az alkalmazás összes rétegét és annak összes stackjét ismeri: 
+
+- Szerver, hálózat, és hosting ismerete
+- alkalmazás skálázhatóságának ismerete 
+- cloud storage ismerete
+- adat redundancia 
+- adat modellek ismerete
+- relációs, nem relációs adatbázisok ismerete
+- üzleti logika ismerete
+- API réteg ismerete
+- MVC pattern ismerete 
+- clean code alkalmazása 
+- user interface átfogó ismerete 
+- modern frameworkok ismerete 
+- user experience 
+- tisztában lenni a végfelhasználó igényeivel
```

Comment:
Ez szerintem eléggé alap dolog, akkor már idejöhetne az is, hogy kommunikáció az ügyféllel, igények egyeztetése, stb. Ha erre gondoltál.